.azure/pipelines/ci.yml

```diff
@@ -38,10 +63,16 @@ jobs:
     steps:
     - powershell: ./eng/scripts/CodeCheck.ps1 -ci
       displayName: Run eng/scripts/CodeCheck.ps1
+    artifacts:
+    - name: Code_Check_Logs
+      path: artifacts/log/
+      publishOnError: true
 
 # Build Windows (x64/x86)
 - template: jobs/default-build.yml
   parameters:
+    dependsOn:
+    - WebJS_Linux_x64_build
```

Comment:
Do we normally want these kinds of dependencies between build jobs?

Reply:
As explained in the other comment, this is needed to flow JS artifacts to Linux Musl builds.